non-protocol-specs/0011-NP-CLIE-client-interaction-tests.md

```diff
@@ -18,3 +18,14 @@ This specification contains a set of tests/acceptance criteria that clients (wal
 6. The parameter `spam.pow.increaseDifficulty` is changed from `1` to `0`.  Verify that
     - This is communicated to the wallet, and wallet uses the new parameter for each transaction tied to a block with a height higher than the one in which the change happened. The wallet should submit no more than `spam.pow.numberofTxPerBlock` PoWs tied to a given block before using another. If this results in none being available the transaction should not be sent. (<a name="0011-NP-CLIE-006" href="#0011-NP-CLIE-006">0011-NP-CLIE-006</a>)
+7. Link more than <SpamPoWNumberOfTxPerBlock>. (currently, 2) transactions to a single block for the PoW; assure that the client does not allow that and the account does not get blocked. (<a name="0011-NP-CLIE-007" href="#0011-NP-CLIE-007">0011-NP-CLIE-007</a>)
+8. Submit significantly (i.e., twice) more than <SpamPoWNumberOfTxPerBlock> * <SpamPowNumberOfPastBlocks> (currently 200) transactions at once. Assure that the client delays or removes the transactions to not exceed the transactions-per-block limit and gets banned.(<a name="0011-NP-CLIE-008" href="#0011-NP-CLIE-008">0011-NP-CLIE-008</a>)
+9. Submit significantly (i.e., twice) more than <SpamPoWNumberOfTxPerBlock> * <SpamPowNumberOfPastBlocks> (currently 200) transactions at once, and keep submitting transactions at a rate of 20tx per second for at least a 10 minutes. Assure that the client delays or removes the transactions to not exceed the transactions-per-block limit and gets banned, and that the client does not crash due to a buffer overload.(<a name="0011-NP-CLIE-009" href="#0011-NP-CLIE-009">0011-NP-CLIE-009</a>)
+10. Submit significantly (ten times) as many votes than allowed (<SpamProtectionMaxVotes>=3) for the same issue at the same time, and verify that the wallet does not allow more than (<SpamProtectionMaxVotes>=3) to be simultaneously, i.e., cause the account to get blocked.(<a name="0011-NP-CLIE-010" href="#0011-NP-CLIE-010">0011-NP-CLIE-010</a>)
+11. Submit significantly (ten times) as many proposals than allowed (<SpamProtectionMaxProposals>=3)at the same time (from an account that has more than <SpamProtectionMinTokensForProposal> tokens), and verify that the wallet does not allow more than (<SpamProtectionMaxProposals>=3) to be simultaneously, i.e., cause the account to get blocked.(<a name="0011-NP-CLIE-011" href="#0011-NP-CLIE-011">0011-NP-CLIE-011</a>)
+12. Submit significantly (ten times) as many delegations than allowed (<SpamProtectionMaxDelegations>=390)at the same time (from an account that has more than <SpamProtectionMinTokensForDelegation> tokens), and verify that the wallet does not allow more than (<SpamProtectionMaxProposals>=390) to be simultaneously, i.e., cause the account to get blocked.(<a name="0011-NP-CLIE-012" href="#0011-NP-CLIE-012">0011-NP-CLIE-012</a>)
```

Comment:
Minor: if I remember correctly, the spam rules for "delegations" include both delegating and undelegating so it might we both throwing at core half of one and half of the other.